### DISCUSSION TITLE
How i store data of all steps in the package of laravel-livewire-wizard spatie

### DISCUSSION THREAD

**Discussion by Programmers12** [spatie/laravel-livewire-wizard#13] [Q&A] [answered]
How i store data of all steps in the package of laravel-livewire-wizard spatie

**Comment by freekmurze** [comment, marked as answer]
Out of the box, the package doesn't store anything. You can get state as mentioned [here](https://spatie.be/docs/laravel-livewire-wizard/v1/usage/accessing-state), and store the retrieved values however you like.
